images/haproxy/Dockerfile

```diff
@@ -15,7 +15,7 @@
 # This image is a haproxy image + minimal config so the container will not exit
 # while we rewrite the config at runtime and signal haproxy to reload.
 
-ARG BASE="registry.k8s.io/build-image/debian-base:bullseye-v1.4.3"
+ARG BASE="registry.k8s.io/build-image/debian-base:bookworm-v1.0.0"
```

Comment:
we should update the distroless base to match

Reply:
There is no distroless debian 12 available yet, I guess we have to wait.

Reply:
Done :+1: